library/psa_crypto.c

```diff
@@ -1227,15 +1227,16 @@ static psa_status_t psa_copy_key_material( const psa_key_slot_t *source,
     buffer_size = PSA_KEY_EXPORT_MAX_SIZE( source->type,
                                            psa_get_key_bits( source ) );
     buffer = mbedtls_calloc( 1, buffer_size );
-    if( buffer == NULL )
+    if( buffer == NULL && buffer_size != 0 )
```

Comment:
The same idiom is present in `psa_save_generated_persistent_key` just above. Is “import/export persistent raw key: 0 byte” failing? If not why?

Reply:
“import/export persistent raw key: 0 byte” isn't failing because it doesn't use `psa_save_generated_persistent_key`. Looking at the tests that do though (PSA generate persistent key), there doesn't appear to be one testing a zero bit raw key.

Adding one causes a failure and will require a bit of change to the test to add it properly.

Reply:
Filed #50 to track this.